test/ruby/scenarios.json

```diff
@@ -13,5 +13,13 @@
         "features": {
             "ruby": {}
         }
+    },
+    "image_and_feature_installs_same_ruby_version": {
+        "image": "mcr.microsoft.com/devcontainers/universal:2.4.0",
```

Comment:
Is it necessary to use the universal image here?  If someone runs these tests locally (or if they're run in the pipeline) this large image will need to be pulled first)

Reply:
This issue only occurs with the universal image, as it uses this Ruby Feature to install a specific version of ruby. 😓 @joshspicer wondering, do you know of any other image which does that? 

I understand the pain point here, but this is the exact same scenario which is currently breaking.

Reply:
The action is [running out of disk space](https://github.com/devcontainers/features/actions/runs/5627383080/job/15249820817#step:4:6294) (this is a manual run with trace logs which surfaced the cause) and failing due to the added test ^

Removing the test, validated locally that it works as expected 👇 : 

![image](https://github.com/devcontainers/features/assets/24955913/aa41f06c-cf3c-467b-97ad-cf56f8a05679)